.changeset/fusion-framework-module-msal-node.next.md

```diff
@@ -0,0 +1,13 @@
+---
+"@equinor/fusion-framework-module-msal-node": major
+---
+
+This release makes `@equinor/fusion-framework-module-msal-node` an explicit ESM package by setting `type: "module"` in `package.json` and updating all internal TypeScript imports to use explicit `.js` extensions. This ensures compatibility with NodeNext module resolution and ESM environments, and aligns the runtime and published output with ESM standards.
```

Comment:
this all seem wrong or is it the the fusion-framework-module.next that is wrong?

Reply:
both file are almost the same :/